test/e2e/network/network_policy.go

```diff
@@ -1785,9 +1804,180 @@ var _ = SIGDescribe("NetworkPolicy [LinuxOnly]", func() {
 	})
 })
 
+var _ = SIGDescribe("NetworkPolicy [Feature:SCTPConnectivity][LinuxOnly]", func() {
+	var service *v1.Service
+	var podServer *v1.Pod
+	var podServerLabelSelector string
+	f := framework.NewDefaultFramework("sctp-network-policy")
+
+	ginkgo.BeforeEach(func() {
+		// Windows does not support network policies.
+		e2eskipper.SkipIfNodeOSDistroIs("windows")
+	})
+
+	ginkgo.Context("NetworkPolicy between server and client", func() {
+		ginkgo.BeforeEach(func() {
+			ginkgo.By("Creating a simple server that serves on port 80 and 81.")
+			podServer, service = createServerPodAndService(f, f.Namespace, "server", []int{80, 81}, []sctpPort{{80, 82}, {81, 83}}...)
```

Comment:
This creates a server that's listening on both TCP and SCTP but you never use the TCP ports